src/common/lib/types/defaultprotocolmessage.ts

```diff
@@ -0,0 +1,10 @@
+import ProtocolMessage, { fromDeserialized } from './protocolmessage';
+
+/**
+ `DefaultPresenceMessage` is the class returned by `DefaultRealtime`’s `PresenceMessage` static property. It exposes the static methods that are used by the tests.
```

Comment:
Is it not possible to just use the `fromDeserialized` function in the tests?

Reply:
How would you imagine the tests getting hold of that function? As things stand, the `fromDeserialized` function is only exported by the modular version of the SDK. Ditto for the `// Used by tests` stuff in `DefaultMessage`.

Reply:
Is it possible to just expose the `fromDeserialized` standalone function in the non-modular version too? that would be my preference

Reply:
Sure thing, have exported it as `protocolMessageFromDeserialized`.